crypto/src/utils.rs

```diff
@@ -0,0 +1,30 @@
+use blsttc::{Error, PublicKeySet, PublicKeyShare, Signature, SignatureShare, PK_SIZE, SIG_SIZE};
+use hex;
+use std::collections::HashMap;
+
+pub fn verify(pk_share: String, msg: &[u8], signature_share: String) -> bool {
```

Comment:
I believe there exists a better abstraction than putting some functions inside keypair and others here. When it comes to good abstraction, one possible criterion would be how convenient it is to use from other crates.

Reply:
I will introduce new `Signature` struct that can be facilitated any messages from the consensus process.